robber/explanation.py

```diff
@@ -0,0 +1,106 @@
+from difflib import Differ
+
+
+class Explanation:
+    """
+    This class gives use a simple way to implement explanations for your expectations.
+    """
+    def __init__(
+            self, a, is_negative, action, b=None, c=None,
+            additional_info=None, negative_action=None, additional_action=None, force_disable_repr=False,
+            need_to_build_diffs=False, z=None
+    ):
+        """
+        
+        :param a: first object
+        :param is_negative: is this a negative explanation?
+        :param action: a string defining an action, ex: "equal", "be True", "called"...
+        :param b: second object
+        :param c: third object
+        :param additional_info: custom message that will be put right after the expectation sentence
+        :param negative_action: the action used in negative explanation, ex: "be False", "exclude"...
+        :param additional_action: the action applied on the third object
+        :param force_disable_repr: force disable the use of repr()
+        :param need_to_build_diffs: trigger if you want to build the diffs string
+        :param z: additional object ex: Expected A to be called with B but actually called with Z 
+        """
+        self.a = a
```

Comment:
I guess a is actual, b is expected and what is c? Can we have a better naming here?